src/services/admin-api.ts

```diff
@@ -14,12 +16,13 @@ export interface AdminApi {
 }
 
 export interface User {
-  id: number;
+  id?: number;
```

Comment:
Should these be required, so we are guaranteed to have a user's info?

Reply:
When creating a new user, there is not yet an ID. So technically it's optional...

https://github.com/condomanagement/condo-web/pull/73/files#diff-69a5cc7b44e72ee82a4e222adc86c224R378

Open to ideas though!

Reply:
Oh okay that's true! 
Hmm I'm not sure about ID, but we can probably add form validation for name/unit/email/phone instead then?

Reply:
Let's add form validation for these as part of the other story to handle better form validation. Phone is optional though.